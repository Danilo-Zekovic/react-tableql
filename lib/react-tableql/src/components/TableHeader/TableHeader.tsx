import React, { FC } from 'react'

const TableHeader: FC = () => {
  return (
    <tr className={`TableQL-tr`}>
      <th scope="col" className={`TableQL-th`}>
        Account
      </th>
      <th scope="col" className={`TableQL-th`}>
        Due Date
      </th>
      <th scope="col" className={`TableQL-th`}>
        Amount
      </th>
      <th scope="col" className={`TableQL-th`}>
        Period
      </th>
    </tr>
  )
}

export default TableHeader
