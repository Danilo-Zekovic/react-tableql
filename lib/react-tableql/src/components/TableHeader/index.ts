import TableHeader from './TableHeader'

export { TableHeader }
