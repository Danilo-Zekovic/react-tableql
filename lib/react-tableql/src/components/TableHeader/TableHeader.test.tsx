import React from 'react'

import { render } from '@testing-library/react'

import TableHeader from './TableHeader'

describe('<TableHeader>', () => {
  it('default snapshot', () => {
    const thead = document.createElement('thead')

    const { container } = render(<TableHeader />, {
      container: document.body.appendChild(thead),
    })

    expect(container.firstChild).toMatchSnapshot()
  })
})
