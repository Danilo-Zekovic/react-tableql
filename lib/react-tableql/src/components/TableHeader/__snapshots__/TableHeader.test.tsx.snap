// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TableHeader> default snapshot 1`] = `
<tr
  class="TableQL-tr"
>
  <th
    class="TableQL-th"
    scope="col"
  >
    Account
  </th>
  <th
    class="TableQL-th"
    scope="col"
  >
    Due Date
  </th>
  <th
    class="TableQL-th"
    scope="col"
  >
    Amount
  </th>
  <th
    class="TableQL-th"
    scope="col"
  >
    Period
  </th>
</tr>
`;
